pages/library/index.js

```diff
@@ -1,129 +1,112 @@
+  position: relative;
 
-const BackgroundImage = styled(Image)`
-  position: absolute;
-  top: 0;
-  left: 0;
-  opacity: 0.7;
-  z-index: -3;
-  object-fit: cover;
+  li {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+  }
 `;
+const BookPageImage = styled.div`
+  position: relative;
+  z-index: -2;
+  height: 100%;
 
-export default function AddWisdom({ handleNewWisdomSubmit }) {
-  const inputRef = useRef(null);
-  const [popupActive, setPopupActive] = useState(false);
-
-  function handleSubmit(event) {
-    event.preventDefault();
+  &::before {
+    content: "";
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-image: url("/assets/bookpage.png");
 
-    const formData = new FormData(event.target);
-    const wisdomData = Object.fromEntries(formData);
-    handleNewWisdomSubmit({ ...wisdomData, answeredRight: 0 });
-    setPopupActive(true);
-    setTimeout(() => {
-      setPopupActive(false);
-    }, 1500);
-    event.target.reset();
-    inputRef.current.focus();
+    background-size: cover;
+    opacity: 0.7;
+    z-index: -1;
   }
-  useEffect(() => {
-    inputRef.current.focus();
-  }, []);
+`;
+export default function ViewBook({ library }) {
+  const categories = library
+    .map((wisdom) => wisdom.category)
+    .filter((value, index, self) => self.indexOf(value) === index);
+
   return (
     <>
-      <BackgroundImage
-        src="/assets/desktop.png"
-        alt="desktop"
-        width="400"
-        height="740"
-      ></BackgroundImage>
-      <StyledBook
-        src="/assets/openbook.png"
-        alt="book"
-        width="1080"
-        height="1920"
-      />
-      <StyledForm onSubmit={handleSubmit}>
-        <StyledLabel htmlFor="question">Enter Question:</StyledLabel>
-        <StyledInput
-          ref={inputRef}
-          required
-          name="question"
-          type="text"
-        ></StyledInput>
-        <StyledLabel htmlFor="answer">Enter Answer:</StyledLabel>
-        <StyledInput required name="answer" type="text"></StyledInput>
-        <StyledLabel htmlFor="category">Pick a Category:</StyledLabel>
-        <StyledSelect name="category">
-          <option value="Vehicles">Vehicles</option>
-          <option value="Food">Food</option>
-          <option value="Basics">Basics</option>
-          <option value="Javascript">Javascript</option>
-        </StyledSelect>
-        <StyledLabel htmlFor="benfit">Pick a Benefit</StyledLabel>
-        <StyledSelect name="benefit">
-          <option value="health">Health</option>
-          <option value="armor">Armor</option>
-          <option value="stamina">Stamina</option>
-        </StyledSelect>
-        <StyledButton type="submit">SUBMIT</StyledButton>
-      </StyledForm>
-      {popupActive && <StyledPopup>Wisdom Added</StyledPopup>}
+      <BookPageContainer>
+        <BookPageImage>
+          {categories.map((cat, index) => {
+            return (
+              <BookPageStyle key={uuidv4()}>
```

Comment:
The categories are unique (based on your filter above) so i think we could use those for the key rather than needing to use `uuidv4`.